napari_plugin_manager/styles.qss

```diff
@@ -0,0 +1,334 @@
+/* ------------ Plugin Dialog ------------ */
```

Comment:
Let's put a little comment here that this must be loaded with `get_current_stylesheet`.

Reply:
Sounds good